DotNETDepends/Program.cs

```diff
@@ -52,9 +58,9 @@ static async Task Main(string[] args)
         }
 
     }
-
-    private static bool validateFileNameInput(string v)
+    private static void PrintVersion()
     {
-        throw new NotImplementedException();
+        //!!When you change this version be sure to update the expected version in dot-net.ts in the codesee repo!!
```

Comment:
This feels like a maintenance pain in the neck. Why do we check the version?

Reply:
It is in the existing code for some reason.  I wouldn't expect an old version to be hanging around in a container, so I'm not sure I get the why? of it.